modules/swagger-codegen/src/main/resources/cpprest/model-header.mustache

```diff
@@ -5,12 +5,13 @@
  * {{description}}
  */
 
-#ifndef {{modelHeaderGuardPrefix}}_{{classname}}_H_
-#define {{modelHeaderGuardPrefix}}_{{classname}}_H_
+#ifndef {{classname}}_H_
```

Comment:
why change this?

Reply:
Copy and paste mistake from an older version of the file, I corrected it